frontend/packages/kubevirt-plugin/integration-tests/tests/vm.wizard.scenario.ts

```diff
@@ -50,14 +56,20 @@ describe('Kubevirt create VM using wizard', () => {
   };
 
   beforeAll(async () => {
-    createResources([multusNAD, testDataVolume]);
+    createResources([testDataVolume]);
   });
 
   afterAll(async () => {
-    deleteResources([multusNAD, testDataVolume]);
+    deleteResources([testDataVolume]);
+  });
+
+  beforeEach(async () => {
+    createResources([multusNAD]);
   });
 
   afterEach(() => {
+    deleteResources([multusNAD]);
```

Comment:
nice to see the issue fix, although you can just use singular calls here like `deleteResource` etc